### PR TITLE
[moveit_ros_benchmarks] Add missing dependency

### PR DESCRIPTION
**Pull request by galou**
Signed-off-by: Gaël Écorchard <gael.ecorchard@cvut.cz>

### Description

`moveit_ros_benchmarks` doesn't compile without `merge-install` without this.

### Checklist
- [x] **Required by CI**: Code is auto formatted using [clang-format](http://moveit.ros.org/documentation/contributing/code)
- [ ] Extend the tutorials / documentation [reference](http://moveit.ros.org/documentation/contributing/)
- [ ] Document API changes relevant to the user in the [MIGRATION.md](https://github.com/ros-planning/moveit/blob/master/MIGRATION.md) notes
- [ ] Create tests, which fail without this PR [reference](https://ros-planning.github.io/moveit_tutorials/doc/tests/tests_tutorial.html)
- [ ] Include a screenshot if changing a GUI
- [ ] While waiting for someone to review your request, please help review [another open pull request](https://github.com/ros-planning/moveit/pulls) to support the maintainers

[//]: # "You can expect a response from a maintainer within 7 days. If you haven't heard anything by then, feel free to ping the thread. Thank you!"

### CODE REVIEW

**Comment by henningkayser** [comment]
> Maybe someone can explain me what's happening there. The stderr is
> 
> ```
> --- stderr: moveit_ros_benchmarks                                                                                                                                                                                   
> /home/libor/robocon_ros2_monows/src/moveit2/moveit2/moveit_ros/benchmarks/src/BenchmarkExecutor.cpp: In member function ‘virtual void moveit_ros_benchmarks::BenchmarkExecutor::writeOutput(const moveit_ros_benchma
> rks::BenchmarkExecutor::BenchmarkRequest&, const string&, double)’:                                                                                                                                                 
> /home/libor/robocon_ros2_monows/src/moveit2/moveit2/moveit_ros/benchmarks/src/BenchmarkExecutor.cpp:1114:31: error: ‘MOVEIT_VERSION_STR’ was not declared in this scope; did you mean ‘MOVEIT_VERSION_EXTRA’?       
>  1114 |   out << "MoveIt version " << MOVEIT_VERSION_STR << '\n';                                                                                                                                                   
>       |                               ^~~~~~~~~~~~~~~~~~                                                                                                                                                            
>       |                               MOVEIT_VERSION_EXTRA                                                                                                                                                          
> make[2]: *** [CMakeFiles/moveit_ros_benchmarks.dir/build.make:76: CMakeFiles/moveit_ros_benchmarks.dir/src/BenchmarkExecutor.cpp.o] Error 1                                                                         
> make[1]: *** [CMakeFiles/Makefile2:109: CMakeFiles/moveit_ros_benchmarks.dir/all] Error 2                                                                                                                           
> make: *** [Makefile:141: all] Error 2
> ```
> 
> As you can see, the compiler doesn't complain about the fact that it didn't find `moveit/version.h`. So, how is it possible that the macro is not defined if the file was found?

This probably happens because of conflicts with an already installed debian version of `moveit_core`, or possibly an underlay (do you have any of those?). The `MOVEIT_VERSION_STR` was just added recently and will not be present in older header files. Btw, adding `moveit_core` is not wrong, but it should also not be necessary as it's already provided and exported with `moveit_ros_planning` which is already included.

**Comment by galou** [comment]
I do have `moveit_core` system package installed and I need it for another workspace, where I'd rather not pull the whole moveit source. Thanks for the explanation!

I'd rather include `moveit_core` as this avoids `moveit_ros_benchmarks` making the assumption that another package includes it.

File: moveit_ros/planning_interface/move_group_interface/src/move_group_interface.cpp [hunk 1]
```diff
@@ -1012,7 +1012,7 @@ class MoveGroupInterface::MoveGroupInterfaceImpl
       std_msgs::msg::String event;
       event.data = "stop";
       trajectory_event_publisher_->publish(event);
-      rclcpp::spin_some(pnode_);
+      // rclcpp::spin_some(pnode_);
```
Comment (JafarAbdi) [inline moveit_ros/planning_interface/move_group_interface/src/move_group_interface.cpp:1015]: I agree that there's no need to spin for publishing, please either remove this line or revert commenting this line

**Review by JafarAbdi — COMMENTED** [review]
Just one request other than that LGTM, multiple headers are being included from moveit_core without having it as a dependency. thanks for the fix

File: moveit_ros/planning_interface/move_group_interface/src/move_group_interface.cpp (hunk 1 above)
Comment (galou) [reply moveit_ros/planning_interface/move_group_interface/src/move_group_interface.cpp]: Ups, the change was not supposed to be committed.
@JafarAbdi this issue with spinning was discussed [there](#1127). I'd like to make a change related to this but I don't know which strategy to choose.

**Review by JafarAbdi — APPROVED** [review]

**Review by henningkayser — APPROVED** [review]

**Pull request merged**